docs/site/Model.md

```diff
@@ -87,12 +87,9 @@ import {model, property} from '@loopback/repository';
 
 @model()
 export class Customer {
-  @property()
-  email: string;
-  @property()
-  isMember: boolean;
-  @property()
-  cart: ShoppingCart;
+  @property() email: string;
+  @property() isMember: boolean;
+  @property() cart: ShoppingCart;
```

Comment:
VSC has applied these code formatting automatically. Is it OK, or should I exclude these changes?

Reply:
Ideally, we should preserve the current style and revert these changes.

Could you please check that you have the latest version of the Prettier plugin for VS Code and that the plugin is configured to use the prettier version from the project? You may want to run `npm run lerna:clean && npm install` to ensure you have the latest version of prettier installed.